lib/minitext/twilio_gateway.rb

```diff
@@ -9,15 +9,15 @@ def initialize(config={})
       token = config[:token]
       subaccount = config[:subaccount]
       @client = Twilio::REST::Client.new(sid, token)
-      @client = client.accounts.find(subaccount) if subaccount
+      @client = client.accounts.get(subaccount) if subaccount
     end
 
     def deliver(message)
-      client.sms.messages.create(
+      client.messages.create(
```

Comment:
Is there a minimum version of the [twilio-ruby](https://github.com/twilio/twilio-ruby) gem required to support this change? If so, we should enforce it in the gemspec.